lambda/utils.js

```diff
@@ -111,6 +111,23 @@ module.exports.waitForFunctionUpdate = async(lambdaARN) => {
     return lambda.waitFor('functionUpdated', params).promise();
 };
 
+module.exports.waitForAliasActive = async(lambdaARN, alias) => {
```

Comment:
For coverage purposes, I would add a simple test for this in test-utils.js.

Similar to the `waitForFunctionUpdate` test:

```

// note: waiters aren't correctly mocked by aws-sdk-mock (for now)
// https://github.com/dwyl/aws-sdk-mock/issues/173
AWS.mock('Lambda', 'waitFor', {});

describe('waitForFunctionUpdate', () => {

        it('should return if LastUpdateStatus is successful', async() => {
            // TODO: remove waitFor mock and test this properly
            await utils.waitForFunctionUpdate('arn:aws:lambda:us-east-1:XXX:function:YYY');
        });

    });
```

Reply:
will do